scripts/myscripts.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){   
+});
+
+//i think you need to create OG variables too?
+var female = prompt("Please enter a female’s first name (1 of 3)");
+
+var uncle = prompt("Please enter a male’s first name (1 of 3)");
+
+var aunt = prompt("Please enter a female’s first name (1 of 3)");
+
+var geofig = prompt("Please enter a female’s first name (1 of 3)");
+
+var state = prompt("Please enter a state.");
+
+var occupation = prompt("Please enter an occupation.");
+
+var color = prompt("Please enter a color");
+
+
+//an array
+var nouns = ["noun1", "noun2", "noun3", "noun4", "noun5"];
+var nouns = prompt("Please enter a noun");
```

Comment:
I see you are taking in values here as needed. You should be doing this inside the document.ready() function. Once  you have the values - you will need to place them into the HTML for them to display. Something like:    $('#female').text(female);
This means you want the element with the id 'female' to have it's text set to the value of the variable female.
I hope this helps. We can talk/look at it more when you stop by campus.